app/src/main/java/com/example/hai/asiantech_intent/model/Users.java

```diff
@@ -0,0 +1,56 @@
+package com.example.hai.asiantech_intent.model;
+
+import java.io.Serializable;
+
+/**
+ * Created by Hai on 6/7/2017.
+ */
+
+public class Users implements Serializable{
```

Comment:
Please change the name of class from `Users` to `User`, and add space after in here `Serializable{`